### ISSUE TITLE
The label for bf:identifies is misspelt/misspelled

### ISSUE THREAD

**Issue by klngwll** [lcnetdev/bibframe-ontology#7]
The label for bf:identifies should probably be "Resource identified".

```
<owl:ObjectProperty rdf:about="http://id.loc.gov/ontologies/bibframe/identifies">
  <rdfs:domain rdf:resource="http://id.loc.gov/ontologies/bibframe/Identifier"/>
  <skos:definition>Resource that is associated with a character string that serves to differentiate one resource from another.</skos:definition>
  <rdfs:label>Resouce identified</rdfs:label>
  <owl:inverseOf rdf:resource="http://id.loc.gov/ontologies/bibframe/identifiedBy"/>
  <dcterms:modified>2017-02-03 (New inverse)</dcterms:modified>
</owl:ObjectProperty>
```

**Comment by jodiw01** [comment]
Issue #10 ("Typo in rdfs:label of bf:identifies") is a duplicate

**Comment by thisismattmiller** [comment]
Fixed with #47 Bibframe 2.0.1
